### PR TITLE
general overhaul of Raspberry installation page

### PR DESCRIPTION
**Pull request by edolis**
This commit is to make more immediate and clear the installation steps for Rasspberry Pi. Currently the openHAB installation pushes the newcomer to read the Linux installation page, which

- starts back from Prerequisites, already covered by the page
- list all the possible configurations for the Linux world. For a newcomer, who potentially does not know anything about Linux, discriminating Debian from the other Linux dialect is frustrating, confusing and unnecessary (as he will unlikely use the infos on old Linux versions or Ubuntu stuff for his RB)
- need to be heavily reformatted for clarity (collapsible do not work, there are no dedicated sections and its always an alternating apt-yum-arch and the like.

Signed-off-by: Emanuele Dolis <edoliscom@yahoo.it>

### CODE REVIEW

File: installation/rasppi.md [hunk 1]
```diff
@@ -79,46 +85,289 @@ Raspbian Lite comes without Java to begin with.
...
+    ```
+   You'll be prompted to select the alternative, pick the zulu11 you installed.
+
+5. If you wish so, you can now delete the downloaded tar file
+    ```shell
+    sudo rm *.tar.gz
+    ```
+
+### Installation of openHAB:
+
```
Comment (Confectrician) [inline installation/rasppi.md:128]: Please remove one blank line.

File: installation/rasppi.md [hunk 2]
```diff
@@ -79,46 +85,289 @@ Raspbian Lite comes without Java to begin with.
...
+| Userdata like rrd4j databases | `/var/lib/openhab`          |                               |
+|         Backups folder        | `/var/lib/openhab/backups`  |                      |
+|     Service configuration     | `/etc/default/openhab`      | (                                 |
+
+
+### Utilities
+
+#### Service Control
+
+
```
Comment (Confectrician) [inline installation/rasppi.md:252]: Please remove one blank line.

**Review by Confectrician — COMMENTED** [review]
Hey @edolis 

Thanks for your contribution.

See my small comments below.

I will do another proofreading round tomorrow and maybe give some additional feedback about the file locations and the new paragraphs.

**Comment by edolis** [comment]
sure... I thought that extra lines were automatically collapsed by GIT.
Since I am very new, what should I do now, perform the editing and commit and pull again? never done before..

File: installation/rasppi.md (hunk 1 above)
Comment (edolis) [reply installation/rasppi.md:128]: do you mean 126? 125 is closing code..

File: installation/rasppi.md [hunk 3]
```diff
@@ -79,46 +85,289 @@ Raspbian Lite comes without Java to begin with.
...
+1. First, add the openHAB Bintray repository key to your package manager and allow Apt to use the HTTPS Protocol:
+
+   ```shell
+   wget -qO - 'https://bintray.com/user/downloadSubjectPublicKey?username=openhab' | sudo apt-key add -
+   sudo apt-get install apt-transport-https
+   ```
+
+2. choose the openHAB package to install. You can choose between *Official (Stable)*, *Beta* or *Snapshot* builds.
+
+    If you are new to openHAB and don't know what to pick, you likely need the *Stable* version. Expand the section of your choice for the details of the installation step.
```
[the remark this answers is not in the document]
Comment (edolis) [reply installation/rasppi.md:142]: Do you mean after "version."correct?

File: installation/rasppi.md [hunk 4]
```diff
@@ -79,46 +85,289 @@ Raspbian Lite comes without Java to begin with.
...
+    sudo update-alternatives --install /usr/bin/java java /opt/jdk/zulu11.43.100-ca-jdk11.0.9.1-linux_aarch32hf/bin/java 1
+    sudo update-alternatives --install /usr/bin/javac javac /opt/jdk/zulu11.43.100-ca-jdk11.0.9.1-linux_aarch32hf/bin/javac 1
+   ```
+6. change the current alternative:
+    ```shell
+    sudo update-alternatives --config java
+    ```
+   You'll be prompted to select the alternative, pick the zulu11 you installed.
+
+5. If you wish so, you can now delete the downloaded tar file
```
Comment (edolis) [inline installation/rasppi.md:122]: numbering to amend

File: installation/rasppi.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Confectrician) [reply installation/rasppi.md:142]: Yes.
We track changes per line/sentence.

The website build will take care of a proper presentation in paragraphs. :)

File: installation/rasppi.md (hunk 1 above)
Comment (Confectrician) [reply installation/rasppi.md:128]: 128 can be removed, since 129 is a blank line too.

**Comment by Confectrician** [comment]
>sure... I thought that extra lines were automatically collapsed by GIT.

Nope. Git is very good in versioning, but very bad in content checking. 🙂 


>Since I am very new, what should I do now, perform the editing and commit and pull again? never done before..
[...]
I can read below "Add more commits by pushing to the main branch on edolis/openhab-docs." -> this means the further commits on this branch will update the pull request directly?
 I'd try but wouldn't like to make some mess which is painful for you to fix.

Yes. You can do additional changes. Make a commit and synchronize that with your fork here in github.
GitHub will recognize the changes and start a new preview build with your latest changes.

I think you are using vscode currently, if i remember correct from the community dco faq article?

**Comment by edolis** [comment]
yep correct, using VS code.
Doing the commit, then. Added a couple of things about logging, aimed at people with zero Linux experience as I am. Please have a look...

**Comment by Confectrician** [comment]
Closing this one because we will continue in #1420

**Pull request closed without merging**